digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1043,6 +995,7 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     roadAddressLinkLikeToApi(projectAddressLink) ++
       (if (projectAddressLink.isSplit)
         Map(
+          "id" -> projectAddressLink.id,
```

Comment:
can be refactored since we have some duplicated code lines,  for e.g. for id, status, reversed, roadNameBlocked..